### PR TITLE
Manue final branch

### PR DESCRIPTION
**Pull request by Coline1186**

### CODE REVIEW

**Review by emmanuelle-c — APPROVED** [review]

File: client/src/components/Formulaire/FormulaireResponsive.css
Comment (Alexandre78R) [inline client/src/components/Formulaire/FormulaireResponsive.css:1]: Delete code dead please 😺

File: client/src/pages/FirstPage/FirstPage.css
Comment (Alexandre78R) [inline client/src/pages/FirstPage/FirstPage.css:1]: Delete code dead please 👍

**Review by Alexandre78R — CHANGES_REQUESTED** [review]

File: client/src/components/Formulaire/FormulaireResponsive.css
Comment (Coline1186) [reply client/src/components/Formulaire/FormulaireResponsive.css:1]: Done

File: client/src/pages/FirstPage/FirstPage.css
Comment (Coline1186) [reply client/src/pages/FirstPage/FirstPage.css:1]: Done

**Review by Alexandre78R — APPROVED** [review]
Good 👍

**Pull request merged**